### PR TITLE
import and export works with jnativehook version 2.1.0

### PR DESCRIPTION
**Pull request by DavidDamke**
Version 2.03 and 2.1.0 do not work on Linux. System crashes after filechooser is being executed

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -71,7 +71,7 @@
     <dependency>
       <groupId>com.1stleg</groupId>
       <artifactId>jnativehook</artifactId>
-      <version>2.1.0</version>
+      <version>2.0.2</version>
```
Comment (Death111) [inline pom.xml:74]: I would not like to downgrade the dependency.

Can you please validate the error also happens, if we would not create an instance of the jnativehook on linux? I think currently we create one on every OS but do not "register listeners" for linux based systems. Maybe not creating an instance already fixes the issue?

Comment (DavidDamke) [reply pom.xml]: if you  do not "register listeners" for linux it works on Java 8 +11  with the latest jnativehook version

**Comment by Death111** [comment]
This branch fixes #108

File: src/main/java/de/doubleslash/keeptime/App.java [hunk 2]
```diff
@@ -218,10 +219,10 @@ private void initialisePopupUI(final Stage primaryStage) throws IOException {
       LOG.debug("Initialising popup UI.");
 
       globalScreenListener = new GlobalScreenListener();
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/App.java:221]: My idea was to put the if(!OS.isLinux()) switch to the creation of the Listener. If we are on linux, we do not need to create an instance of it :)
- So the change in the GlobalScreenListener class would not be needed.
- The changes in this class would need to validate globalScreenListner!=null insted of the linux switch.

File: src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java [hunk 3]
```diff
@@ -28,6 +28,7 @@
 public class DateFormatterTest {
 
    @Test
+   @Disabled
```
Comment (Death111) [inline src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java:31]: why do we need to disable it now?

**Review by Death111 — CHANGES_REQUESTED** [review]

File: src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java (hunk 3 above)
Comment (DavidDamke) [reply src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java:31]: I thought it got executed although the class was Disabled.  But I tested it and now it works again

File: src/main/java/de/doubleslash/keeptime/App.java (hunk 2 above)
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/App.java]: reverted the changes in GlobalScreenListener

**Review by Death111 — APPROVED** [review]

**Pull request merged**